include/hpp/core/time-parameterization/polynomial.hh

```diff
@@ -102,6 +108,7 @@ namespace hpp {
               tn *= t;
               res += a[i] * tn;
             }
+            assert (!std::isnan(res));
```

Comment:
This is C++ >= 11. Can you enclose it within:
```C++
#if __cplusplus__ >= 201103L
#endif
```

Reply:
Alternatively, you can use `res == res` which fails only if res is a nan.

Reply:
I did it and force pushed.